src/test/java/com/ea/restaurant/service/impl/Oauth2ServiceImplTest.java

```diff
@@ -54,7 +59,8 @@ public void setUp() throws JOSEException {
     appClientScope = AppClientScopeFixture.buildAppClientScope(1L);
     appRefreshToken = AppRefreshTokenFixture.buildAppRefreshToken(1L);
 
-    var mockedOauth2Util = Mockito.mockStatic(Oauth2Util.class);
+    mockedOauth2Util = Mockito.mockStatic(Oauth2Util.class);
```

Comment:
Why not to just mock this class where you need it, instead of creating a new variable and adding new mock call like `this.mockedOauth2Util.close();`